reconcile/utils/runtime/environment.py

```diff
@@ -19,7 +19,9 @@
 
 
 def init_env(
-    log_level: Optional[str] = None, config_file: Optional[str] = None
+    log_level: Optional[str] = None,
+    config_file: Optional[str] = None,
+    print_gql_url: Optional[bool] = True,
```

Comment:
Why is `print_gql_url` optional? `True`, `False`, `None`?

Reply:
🤦  thank you - fixed